qsignature/src/org/qcmg/sig/SignatureCompareRelatedSimple.java

```diff
@@ -38,6 +37,7 @@
  *  
  * @author o.holmes
  *
+ *@deprecated Superseded by Compare
```

Comment:
again, this class should be deprecated now and delete it on next version.

Reply:
done